File: responses/3JJAY9-8X9/count_of_largest_integer.js
// you can write to stdout for debugging purposes, e.g.
// console.log('this is a debug message');

function solution(A) {
    var maxEle=A[0];
    var count =1;
    var size=A.length;
    for(var i=1;i<size;i++){
        if(A[i]>maxEle){
            maxEle=A[i];
            count = 1;
        }
        else if(A[i]==maxEle){
            count++;
        }
    }
    if(size>0){
        return count;
    }
    else{
        return 0;
    }

}

File: responses/EC4GJT-SMK/count_of_largest_integer.py
# you can write to stdout for debugging purposes, e.g.
# print("this is a debug message")

def solution(A):
    # write your code in Python
    highest = A[0]
    count = 0
    for item in A:
        if item == highest:
            count += 1
        elif item > highest:
            highest = item
            count = 1
        else:
            pass
    return count
